pynvjitlink/_nvjitlinklib.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+# A shim layer to use new bindings from CUDA Python
+from cuda.bindings import nvjitlink
+from cuda.bindings.nvjitlink import nvJitLinkError
+
+
+def nvjitlink_version():
```

Comment:
This is kind of an odd looking module.  Is it provided just for backward compatibility?  Also, why is this one method not just called `version()`?

Reply:
> This is kind of an odd looking module. Is it provided just for backward compatibility?

Yup! The whole purpose of this module is to kill the old one (that was implemented using CPython C APIs). This method replaces the following one:
https://github.com/rapidsai/pynvjitlink/blob/666375a8696c260b1431f5292e4413516af8e706/pynvjitlink/_nvjitlinklib.cpp#L58